README.md

```diff
@@ -45,7 +45,18 @@ For more info on postgres commands, see https://www.postgresql.org/docs/
 
 ## Usage Instructions
 
-TODO: write these
+### Seed the catalog
+
+Without profiling:
+```ruby
+RAILS_ENVIRONMENT=production bundle exec rake seed_catalog
+```
+
+With profiling:
+```ruby
+RAILS_ENVIRONMENT=production bundle exec rake seed_catalog[profile]
```

Comment:
is this actually what we type?  `seed_catalog[profile]`  ?  or do we do some string substitution for "profile" ?

Reply:
Yes, you literally type `seed_catalog[profile]`, no substitution.